.github/workflows/melos_bootstrap.yaml

```diff
@@ -0,0 +1,24 @@
+name: "Melos bootstrap"
+on: [pull_request]
+
+permissions:
+  contents: read
+
+jobs:
+  melos-bootstrap:
+    strategy:
+      fail-fast: false
+      matrix:
+        args:
+          - ""
+          - '--ignore="at_root_server" --ignore="at_secondary"'
```

Comment:
This is failing as it's `at_secondary_server`

Reply:
Actually that's correct, it needs, the package name and in this case the package name doesn't match the directory name.

Not sure why the action is failing... it couldn't find the trunk branch of at_demo_data...